inference-engine/src/inference_engine/ie_compound_blob.cpp

```diff
@@ -192,6 +192,50 @@ void verifyI420BlobInput(const Blob::Ptr& y, const Blob::Ptr& u, const Blob::Ptr
     }
 }
 
+TensorDesc verifyBatchedBlobInput(const std::vector<Blob::Ptr>& blobs) {
+    // verify invariants
+    if (blobs.size() < 1) {
+        THROW_IE_EXCEPTION << "at least 1 sub-blob needed";
```

Comment:
Can we extend error message?

Reply:
Thank you.
Can you please help me understand, how this error message to be extended?

Reply:
> Can you please help me understand, how this error message to be extended?

`BatchedBlob cannot be created from empty vector of Blob, Please, make sure vector contains at least one Blob`

Reply:
done